ejb3/src/main/java/org/jboss/as/ejb3/timerservice/TimerHandleImpl.java

```diff
@@ -21,115 +21,72 @@
  */
 package org.jboss.as.ejb3.timerservice;
 
-import java.security.AccessController;
-
-import javax.ejb.EJBException;
-import javax.ejb.Timer;
 import javax.ejb.TimerHandle;
 
+import org.jboss.as.ejb3.component.EJBComponent;
 import org.jboss.as.ejb3.logging.EjbLogger;
+import org.jboss.as.ejb3.timerservice.spi.ManagedTimer;
 import org.jboss.as.server.CurrentServiceContainer;
-import org.jboss.msc.service.ServiceContainer;
 import org.jboss.msc.service.ServiceName;
+import org.wildfly.security.manager.WildFlySecurityManager;
 
 /**
+ * Serializable handle for an EJB timer.
  * @author <a href="mailto:cdewolf@redhat.com">Carlo de Wolf</a>
- * @version $Revision: $
+ * @author Paul Ferraro
  */
 public class TimerHandleImpl implements TimerHandle {
     private static final long serialVersionUID = 1L;
 
-    /**
-     * Id of the target {@link org.jboss.as.ejb3.timerservice.spi.TimedObjectInvoker}
-     */
-    private final String timedObjectId;
-
-    /**
-     * The service name of the timer service
-     */
-    private final String serviceName;
-
-    /**
-     * Each {@link org.jboss.as.ejb3.timerservice.spi.TimedObjectInvoker} can have multiple timer instances.
-     * This id corresponds to one such <i>instance</i>
-     */
     private final String id;
+    private final String componentServiceName;
 
```

Comment:
here we're adding 1 field and deleting 2, which breaks serialization compatibility. Existing timer handle will not be properly deserialized with the default serialization.

Reply:
Good point.  I'll make sure to restore serialization compatibility.